code/datums/craft/step.dm

```diff
@@ -121,7 +121,7 @@
 			return
 		if(target)
 			announce_action(start_msg, user, I, target)
-		if(!I.use_tool(user, target || user, time, reqed_quality))
```

Comment:
Crafting now uses the sum of cog and mec stats

We don't have variable difficulty for craft recipes yet, but its a step forward